modules/users/client/views/invite.client.view.html

```diff
@@ -26,6 +26,33 @@
           Codes are valid for 24 hours.
         </p>
 
+        <!-- Go to www.addthis.com/dashboard to customize your tools -->
+        <p>
+          <div ng-if="::invite.getCode() !== ''" >
+          <script type="text/javascript">
+            var code = document.getElementById("invite").value.toUpperCase();
+            var addthis_config = addthis_config||{};
+                addthis_config.ui_email_from = 'help@trustroots.org';
+                addthis_config.ui_email_note = 'Your invite code: ' + code + ' would be shared which can be used for signup with Trustroots at https://www.trustroots.org/signup?code=' + code;
+
+            var addthis_share = {
+              url: 'https://www.trustroots.org/signup?code=' + code,
+              title: 'Invite by ' + user.displayName + ' to join Trustroots',
+              description: 'You have been invited to join Trustroots, an invite only community for travelers. Your invite code to join Trustroots is: ' + code,
+              email_template: 'new_template',
+              email_vars: { 'code': code, 'invitee': user.displayName },
+              passthrough: {
+                twitter: {
+                  text: 'You have been invited to join Trustroots. Your invite code: ' + code + ' and url:'
+                }
+              }
+            };
+          </script>
+          </div>
+          <div class="addthis_inline_share_toolbox"></div>
+          <script type="text/javascript" src="//s7.addthis.com/js/300/addthis_widget.js#pubid=ra-591c2040c6459c4a" async="async"></script>
```

Comment:
Make sure to add `*.addthis.com` to CSP rules: https://github.com/Trustroots/trustroots/blob/13ff4ddf171c09f9224162f9714b85e39e58995f/config/lib/express.js#L270-L271